build/scripts/run_perf.ps1

```diff
@@ -2,6 +2,8 @@ set-variable -name LastExitCode 0
 set-strictmode -version 2.0
 $ErrorActionPreference="Stop"
 
+./build/scripts/cleanup_perf.ps1
+
 Invoke-WebRequest -Uri http://dotnetci.blob.core.windows.net/roslyn-perf/cpc.zip -OutFile cpc.zip
 [Reflection.Assembly]::LoadWithPartialName('System.IO.Compression.FileSystem') | Out-Null
 If (Test-Path C:/CPC) {
```

Comment:
Another hard coded use of CPC location.  we should make it a constant here and pass down to the cleanup_perf.ps1 script